commands/droplets.go

```diff
@@ -475,19 +475,39 @@ func RunDropletDelete(c *CmdConfig) error {
 		return nil
 	}
 
-	if force || AskForConfirm("delete droplet(s)") == nil {
-
-		fn := func(ids []int) error {
+	fn := func(ids []int) error {
+		if !force {
+		
+			// If we aren't forcing, print out the list
+			fmt.Printf("Following Droplet(s) will be deleted:\n")
```

Comment:
Maybe a tiny nitpick, but this could be changed with `Println` (in that case `\n` is not needed).